Ports/gcc/patches/gcc-support-mold-linker.patch

```diff
@@ -0,0 +1,129 @@
+From 37f6a0c24bb07efb29a2222d984e4d793c165aef Mon Sep 17 00:00:00 2001
```

Comment:
Thank you for using a sensible split patch instead of shoving it into the other file.